docs/faq/troubleshooting/why-arent-duplication-metrics-being-calculated.md

```diff
@@ -10,7 +10,7 @@ When this happens, Codacy doesn't display code duplication metrics for the affec
 
     ![Duplication in the Files page](images/duplication-files.png)
 
--   The **Commits** and **Pull Request** pages display an empty **New Duplication** tab.
+-   The **Commits** and **Pull Request** pages display an empty **Duplication** tab.
```

Comment:
Here, I'm assuming that the "duplication" tab itself will be empty, since [earlier in the the page](https://ala-643-doc-update-documentation-fo--docs-codacy.netlify.app/faq/troubleshooting/why-arent-duplication-metrics-being-calculated/#why-arent-duplication-metrics-being-calculated), we imply that "Codacy skips the calculation of code duplication"